README.md

```diff
@@ -1,84 +1,98 @@
-# pikdum's steam deck tools
+[![release-please](https://github.com/SirStig/Steam-deck-tests/actions/workflows/release-please.yaml/badge.svg)](https://github.com/SirStig/Steam-deck-tests/actions/workflows/release-please.yaml)
+
+# Pikdum's Steam Deck Tools
```

Comment:
**thought:** Lowercase was more a stylistic choice, but this is fine.

I'll probably want to tweak some things though.

Reply:
Feel free to tweak anything!